modules/api/pkg/resource/pod/detail_test.go

```diff
@@ -161,7 +161,7 @@ func TestEvalEnvFrom(t *testing.T) {
 		{
 			container: v1.Container{
 				Name:  "echoserver",
-				Image: "k8s.gcr.io/echoserver",
+				Image: "registry.k8s.io/echoserver",
```

Comment:
It doesn't have latest version available, but it's only for testing so we can keep it like it is.